manifests/c/com/exelearning/editor/2.7/com.exelearning.editor.locale.es-ES.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: com.exelearning.editor
+PackageVersion: 2.7
+PackageLocale: es-ES
+Publisher: eXelearning

```

```suggestion
Publisher: INTEF-eXe Project
PublisherUrl: https://exelearning.net
PublisherSupportUrl: https://exelearning.net/help/
```
`Publisher` field must match Control Panel / Apps&Features. Also suggested some more metadata.